{{cookiecutter.repo_name}}/Makefile

```diff
@@ -27,7 +32,14 @@ help:
 clean:
 	find . -name '*.pyc' -delete
 	coverage erase
-	rm -rf assets
+	rm -rf {{cookiecutter.repo_name}}/assets {{cookiecutter.repo_name}}/static/build coverage htmlcov
+
+requirements.js:
+	npm install
+	$(NODE_BIN)/bower install
+
+requirements: requirements.js
```

Comment:
Combine this with local-requirements. I'll deal with local vs. prod later since it will involve a configuration change.